gulpfile.js

```diff
@@ -770,7 +770,7 @@ var generateTable = function(schema)
 {
     var table = "";
 
-    table += "<table class='table table-bordered table-responsive table-hover table-condensed'>";
+    table += "<table class='table table-responsive table-condensed'>";
```

Comment:
I think this code is generating API docs and not the templates used in alpaca to generate forms.